cmd/manager/main.go

```diff
@@ -187,7 +188,10 @@ func main() {
 			}
 		}
 	}
-
+	if err = routev1.AddToScheme(mgr.GetScheme()); err != nil {
```

Comment:
not sure if we should do this, can. Could you please elaborate on why this might be needed?

Reply:
need to fetch the routes to update the isvc status with the correct URL since the route is created by ODH Model Controller. So we have to add the route api to the scheme

Reply:
any resource can be fetched without it, still not sure why it is really needed.